exercises/isogram/isogram_test.rb

```diff
@@ -4,66 +4,66 @@
 require 'minitest/autorun'
 require_relative 'isogram'
 
-# Test data version:
-# 306975e
+# Common test data version: 3dfde31
 class IsogramTest < Minitest::Test
   def test_duplicates
-    str = 'duplicates'
-    assert Isogram.is_isogram?(str)
+    
```

Comment:
If only there was some kind of automatic tool that would detect such things...
&lt;/smartassmode>